src/analysis/retail/monk/mistweaver/modules/spells/PoolOfMists.tsx

```diff
@@ -0,0 +1,140 @@
+
+    this.addEventListener(
+      Events.applybuff.by(SELECTED_PLAYER).spell(SPELLS.RENEWING_MIST_HEAL),
+      this.onApplyRem,
+    );
+
+    this.addEventListener(
+      Events.cast.by(SELECTED_PLAYER).spell(TALENTS_MONK.RISING_SUN_KICK_TALENT),
+      this.onRisingSunKick,
+    );
+  }
+
+  private onApplyRem(event: ApplyBuffEvent) {
+    const playerId = event.targetID;
+    const spellId = SPELLS.RENEWING_MIST_HEAL.id;
+
+    if (!this.hotTracker.hasHot(event, spellId)) {
+      return;
+    }
+    const hot = this.hotTracker.hots[playerId][spellId];
+    if (
+      (this.hotTracker.fromRapidDiffusion(hot) || this.hotTracker.fromHardcast(hot)) &&
+      !this.hotTracker.fromBounce(hot)
+    ) {
+      this.rskCDR += POOL_OF_MISTS_CDR;
+      if (this.spellUsable.isOnCooldown(TALENTS_MONK.RISING_SUN_KICK_TALENT.id)) {
+        const reduction = this.spellUsable.reduceCooldown(
+          TALENTS_MONK.RISING_SUN_KICK_TALENT.id,
+          POOL_OF_MISTS_CDR,
+        );
+        this.rskEffective += reduction;
+        this.rskWaste += POOL_OF_MISTS_CDR - reduction;
+      } else {
+        this.rskWaste += POOL_OF_MISTS_CDR;
+      }
+    }
+  }
+
+  private onRisingSunKick(event: CastEvent) {
+    this.remCDR += POOL_OF_MISTS_CDR;
+    if (this.spellUsable.isOnCooldown(TALENTS_MONK.RENEWING_MIST_TALENT.id)) {
+      const reduction = this.spellUsable.reduceCooldown(
+        TALENTS_MONK.RENEWING_MIST_TALENT.id,
+        POOL_OF_MISTS_CDR,
+      );
+      this.remEffective += reduction;
+      this.remWaste += POOL_OF_MISTS_CDR - reduction;
+    } else {
+      this.remWaste += POOL_OF_MISTS_CDR;
+    }
+  }
+
+  statistic() {
+    return (
+      <Statistic
+        position={STATISTIC_ORDER.CORE(11)}
+        category={STATISTIC_CATEGORY.TALENTS}
+        size="flexible"
+        tooltip={
+          <ul>
+            <li>
+              Total <SpellLink spell={TALENTS_MONK.RENEWING_MIST_TALENT} /> reduction:{' '}
+              {formatDuration(this.remCDR)} ({formatDuration(this.remWaste)} wasted)
+            </li>
+            <ul>
+              <li>{formatNumber(this.extraReMCasts)} additional casts</li>
+            </ul>
+            <li>
+              Total <SpellLink spell={TALENTS_MONK.RISING_SUN_KICK_TALENT} /> reduction:{' '}
+              {formatDuration(this.rskCDR)} ({formatDuration(this.rskWaste)} wasted)
+            </li>
+            <ul>
+              <li>{formatNumber(this.extraRSKCasts)} additional casts</li>
+            </ul>
+          </ul>
+        }
+      >
+        <TalentSpellText talent={TALENTS_MONK.POOL_OF_MISTS_TALENT}>
+          <SpellIcon spell={TALENTS_MONK.RENEWING_MIST_TALENT} />{' '}
+          <ItemCooldownReduction effective={this.remEffective} />
+          <div></div>
+          <SpellIcon spell={TALENTS_MONK.RISING_SUN_KICK_TALENT} />{' '}
+          <ItemCooldownReduction effective={this.rskEffective} />
+        </TalentSpellText>
```

Comment:
Better to put each "line" into its own div instead of using a div as a line break